src/Microsoft.AspNetCore.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -341,7 +359,41 @@ private void BuildFallbackBlock(TagHelperContent builder)
                 .AppendHtml("\",");
 
             AppendFallbackHrefs(builder, fallbackHrefs);
-            builder.AppendHtml("</script>");
+
+            builder.AppendHtml(", \"");
+
+            var relAdded = false;
+
+            // Perf: Avoid allocating enumerator
+            for (var i = 0; i < attributes.Count; i++)
+            {
+                var attribute = attributes[i];
+                if (string.Equals(attribute.Name, HrefAttributeName, StringComparison.OrdinalIgnoreCase))
+                {
+                    continue;
+                }
+
+                if (string.Equals(attribute.Name, RelAttributeName, StringComparison.OrdinalIgnoreCase))
+                {
+                    relAdded = true;
+                }
+
+                attribute.WriteTo(StringWriter, HtmlEncoder);
+                StringWriter.Write(' ');
+            }
+
+            if (!relAdded)
+            {
+                RelAttribute.WriteTo(StringWriter, HtmlEncoder);
+            }
```

Comment:
Huh? Why add a default `rel` attribute? This is a general `<link>` tag helper, not specific to style sheets.